### PR TITLE
Update the host containers for 1.19.3

### PR DESCRIPTION
**Pull request by ytsssun**
**Description of changes:**
Update the admin and control host containers to bottlerocket-admin-container v0.11.6 and bottlerocket-control-container v0.7.10. Also add migrations move to these versions on upgrade.

**Testing done:**

- [x] `cargo make check-migrations`
- [x] Build Bottlerocket AMI and check the version via API Client
```
apiclient get settings.host-containers
{
  "settings": {
    "host-containers": {
      "admin": {
        "enabled": false,
        "source": "328549459982.dkr.ecr.us-west-2.amazonaws.com/bottlerocket-admin:v0.11.6",
        "superpowered": true,
        "user-data": "xxxxxxxxxxxx"},
      "control": {
        "enabled": true,
        "source": "328549459982.dkr.ecr.us-west-2.amazonaws.com/bottlerocket-control:v0.7.10",
        "superpowered": false
      }
    }
  }
}
```

- [x] Test migration from `v1.19.2` to `v1.19.3`, host container versions are correctly picked up. (Same output as above)
- [x] Test rollback to `v1.19.2` from `v1.19.3`, host container were correctly pointing to the older version:
```
[ec2-user@admin]$ apiclient get settings.host-containers
{
  "settings": {
    "host-containers": {
      "admin": {
        "enabled": true,
        "source": "328549459982.dkr.ecr.us-west-2.amazonaws.com/bottlerocket-admin:v0.11.4",
        "superpowered": true,
        "user-data": "eyJzc2giOnsiYXV0aG9yaXplZC1rZXlzIjpbInNzaC1yc2EgQUFBQUIzTnphQzF5YzJFQUFBQURBUUFCQUFBQkFRREN3dWVnVFhTa29KZmpDZm9leDdRLytLTGJWNllVaXdiczFrbVJwc3grdDFEbGFsdS9ZM0VBZDN5TmdKRkdQZ29FZWptejdSaTRZcTZWUDQvWFJoaVl1Z09HNXpmaWtuZ1orMW9lbkppVVRpdDhpeXFYcXAxYTg0T3lWUVllcm5GZkNDWXJtdVFmcmp1d3RiY2VYK3RtS0xqbGtIcjNPVXFjdUxCeXBMSzhaRlVybjVTNUQ1b1EvV2VWeEhiM2NWdW5tVDlOblN0ZU5lZXJtZjVZNXBkMFNQWlNLL3lIVnNqYSttUGVmNFloNVM5YnJqM3hlZWg0THplNXJqQS9OaGpTU2lLVm5oVko2SktkUGhNQ0grM2h6d21Gd3dHK1FiL0VFcFZJSkU4ckl4R0czRWZRQ1I4a2M0OFZqc2tabjlVdzJpOUo2SDdNcHgxbThzdEpuYWVYIHl1dG9uZ3MtMjAyMyJdfX0="
      },
      "control": {
        "enabled": true,
        "source": "328549459982.dkr.ecr.us-west-2.amazonaws.com/bottlerocket-control:v0.7.8",
        "superpowered": false
      }
    }
  }
}
```

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

**Review by gthao313 — APPROVED** [review]

**Comment by bcressey** [comment]
Can you fold the `cargo fmt` changes into the corresponding commits that added the migrations?

File: sources/api/migration/migrations/v1.19.3/aws-admin-container-v0-11-6/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+edition = "2021"
+publish = false
+# Don't rebuild crate just because of changes to README.
+exclude = ["README.md"]
+
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+migration-helpers = { path = "../../../migration-helpers", version = "0.1.0"}
```
Comment (bcressey) [inline sources/api/migration/migrations/v1.19.3/aws-admin-container-v0-11-6/Cargo.toml]: Please configure your editor to save files with a newline at the end.

For VS Code there is this:
https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files

(Note, this is a different behavior than manually adding a blank line at the end of the file - we don't want to do that.)

Comment (bcressey) [reply sources/api/migration/migrations/v1.19.3/aws-admin-container-v0-11-6/Cargo.toml]: All of the new `Cargo.toml` files have the missing-newline problem.

Comment (ytsssun) [reply sources/api/migration/migrations/v1.19.3/aws-admin-container-v0-11-6/Cargo.toml]: Thanks for the link, updated the commits with the newline setup.

**Comment by ytsssun** [comment]
> Can you fold the `cargo fmt` changes into the corresponding commits that added the migrations?

Yes, done.

**Review by gthao313 — APPROVED** [review]

**Review by bcressey — APPROVED** [review]

**Pull request merged**